vllm/entrypoints/openai/conversation.py

```diff
@@ -0,0 +1,739 @@
+# From https://github.com/lm-sys/FastChat/blob/main/fastchat/conversation.py
+"""
+Conversation prompt templates.
+"""
```

Comment:
It is better to import instead of copy. Otherwise, we have to maintain two separate versions of the code.

Reply:
done